icons/supabase.svg

```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Supabase</title><path d="M21.362 9.354H12V.396a.396.396 0 0 0-.716-.233L2.203 12.424l-.401.562a1.04 1.04 0 0 0 .836 1.659H12v8.959a.396.396 0 0 0 .716.233l9.081-12.261.401-.562a1.04 1.04 0 0 0-.836-1.66z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Supabase</title><path d="M11.9925 15.4495s-7.41792-.0197-9.58941.0098c-1.37891.0188-2.121541-1.3022-2.086558-2.0935.034849-.7882.336538-1.1521.56419-1.4376C1.98747 10.5403 9.7708.733045 10.0033.43219c.2783-.3600834.578-.42286158.9104-.42249002.5694.00063652.9785.58089502.9802.96149402.0092 2.095106.0507 7.573926.0507 7.573926s9.3704-.00117 9.8064.01307c1.1568.03778 1.8149 1.11527 1.8865 1.68721.0219.1754.1826.9291-.3642 1.6248-2.5597 3.2567-9.0729 11.4572-9.1807 11.5801-.1616.1844-.3602.5549-1.0044.543-.6785-.0126-.9782-.6694-.9818-.8802-.0355-2.0676-.1139-7.6636-.1139-7.6636Z"/></svg>
```

Comment:
Could you remove the newline at the end of the file?

Reply:
I don't see any newline at the end of the file.

Reply:
never mind I found it.